lib/routes/account.js

```diff
@@ -974,6 +974,31 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
           }
         }
 
+        function sendVerifyLoginPush(devices) {
+          if (doSigninConfirmation) {
+            log.info({
+              op: 'account.signin.confirm.push.start',
+              uid: accountRecord.uid,
+              tokenVerificationId: tokenVerificationId
+            })
+
+            return push.notifyVerifyLogin(accountRecord.uid, devices)
+          }
+        }
+
+        function sendVerifyLogin() {
+          return db.devices(accountRecord.uid).then(
+            function (devices) {
+              //filter the devices variable to devices which can receive a push notification
```

Comment:
@seanmonstar Need to filter this to devices that can receive a push notification (i.e. Firefox 56+ for now). Any pointers on how to do that?